src/styles.scss

```diff
@@ -2,3 +2,82 @@
 
 /* Importing Bootstrap SCSS file. */
 @import 'bootstrap/scss/bootstrap';
+@import '~@angular/material/prebuilt-themes/indigo-pink.css';
+@import 'app/styles/components/button.scss';
+@import 'app/styles/layout/header.scss';
+
+html, body { height: 100%; }
+body { margin: 0;
+    background-color: white;
+}
+
+//h1 styling
+h1{
```

Comment:
Maintain consistency in spacing.
h1{ -> h1 {

follow this to all styles

Reply:
Changes are done